documentation/modules/setup/partials/minikube-setup.adoc

```diff
@@ -300,7 +299,7 @@ Configure Knative to use the `kourier-ingress` Gateway:
 [.console-input]
 [source,bash,subs="+macros,+attributes"]
 ----
-ksvc_domain="\"data\":{\""$(minikube ip)".nip.io\": \"\"}"
+ksvc_domain="\"data\":{\""$(minikube ip -p knativetutorial)".nip.io\": \"\"}"
```

Comment:
With the `minikube` version suggested in the document, I had to pass the profile to the `ip` subcommand.